data/opensource/src/commonMain/kotlin/app/tivi/data/opensource/OpenSourceState.kt

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023, Google LLC, Christopher Banes and the Tivi project contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package app.tivi.data.opensource
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.emptyFlow
+import kotlinx.serialization.Serializable
+
+interface OpenSourceState {
+    val openSourceList: Flow<List<OpenSourceItem>>
+    fun serializeToJson(): String
```

Comment:
What is this used for?

Reply:
sorry for that. This code is writing by following the code style in other module.
I have change it.